tools/ci/run-canary.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
can you enable the shellcheck intellij plugin? it will point out a few things to tweak

Reply:
but TBH, I would go full meta and write this with the Rust SDK :-)

Reply:
It's actually a pretty good exercise of the Rust SDK in addition to the canary Lambda, so I'm onboard with this.